src/senselab/audio/tasks/text_to_speech/huggingface.py

```diff
@@ -49,6 +49,7 @@ def synthesize_texts_with_transformers(
         texts: List[str],
         model: HFModel = HFModel(path_or_uri="suno/bark", revision="main"),
         device: Optional[DeviceType] = None,
+        forward_params: Optional[Dict[str, Any]] = None,
```

Comment:
how do these differ from simple **kwargs?

Reply:
not really different. if you want, i can make it with `**kwargs`